app.rb

```diff
@@ -7,20 +9,37 @@
 require_relative './nameable'
 require_relative './book_selection'
 require_relative './person_selection'
+require_relative './file_handler'
+require_relative './load_people'
+require_relative './load_rentals'
+require_relative './load_books'
 
 class App
   def initialize
     @books = []
     @people = []
     @rentals = []
+    @save_student = []
+    @save_teachers = []
+    @save_rentals = []
+    @save_books = []
     @book_selection = BookSelection.new(@books)
     @person_selection = PersonSelection.new(@people)
+
+    load_data
   end
 
   def create_book
     book_data = prompt_book_data
     book = Book.new(*book_data)
     @books << book
+    @file_handler = FileHandler.new('./book.json')
+    @save_student << {
+      'Title' => book.title,
+      'Author' => book.author,
+      'id' => book.id
+    }
+    @file_handler.write_to_file(@save_student.to_json)
     puts "Created #{book.title} by #{book.author}"
   end
```

Comment:
- [ ] I noticed that the first time I create a `Book` instance, the `Book` instance is saved along with the first `Student` instance that I created. They are saved in the _**JSON**_ file that stores book data. Kindly ensure that it is only `Book` instances that are saved to the JSON file that ought to store book data. 👍